jormungandr/Cargo.toml

```diff
@@ -31,6 +31,7 @@ imhamt = { path = "../chain-deps/imhamt" }
 error-chain = "0.12"
 futures    = "0.1"
 futures03 = { package = "futures", version = "0.3.4", features = ["compat"] }
+futures-util = "0.3"
```

Comment:
There should not be a need to use this directly, it's reexported from `futures03`

Reply:
thanks, it took me a while to find in the first place in future_util